lib/ipam/ipam.go

```diff
@@ -787,13 +787,13 @@ func (c ipamClient) decrementHandle(ctx context.Context, handleID string, blockC
 	for i := 0; i < ipamEtcdRetries; i++ {
 		obj, err := c.client.Get(ctx, model.IPAMHandleKey{HandleID: handleID}, "")
 		if err != nil {
-			log.Fatalf("Can't decrement block because it doesn't exist")
+			log.Panicf("Can't decrement block because it doesn't exist")
```

Comment:
Orthogonal to this PR, but surely we should return an error in this case, not panic?